sdk/textanalytics/azure-ai-textanalytics/swagger/README.md

```diff
@@ -21,29 +27,35 @@ cd <swagger-folder>
 autorest
 ```
 
-Alternately, for using the specific autorest java version, use `--use:@autorest/java@{specific-version}`. The version 
-will be downloaded and compiled locally in `AUTOREST_HOME`.
-
-```ps
-cd <swagger-folder>
-autorest --java --use:@autorest/java@4.1.17 README.md
-```
-
-### Code generation settings
-``` yaml
-use: '@autorest/java@4.1.17'
+### Code Generation
+```yaml
+use: '@autorest/java@4.1.16'
```

Comment:
Any reason to change the use version from `4.1.17` to `4.1.16`? Did you find a bug in the newer version?

Reply:
4.1.17 works fine. I can change back to it.